scripts/create-environment.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+
+echo "OSMOSISD_ENVIRONMENT=mainnet" > $HOME/.osmosisd/.env
```

Comment:
have you tested what happens if `$HOME/.osmosisd/` does not exist?

Reply:
Yes we should check if there was an error in the previous install step